examples/build.rs

```diff
@@ -1,14 +1,11 @@
 fn main() {
     if cfg!(feature = "esp32c6") || cfg!(feature = "esp32h2") {
+        // TODO this shouldn't be required to be done by the user but done in `linkall.x`
```

Comment:
Is this documented somewhere? I'd prefer resolving this TODO in this PR if not.

Reply:
I was about to create an issue for it .... but on the other hand, this PR isn't too complex and adding it probably is fine and pretty much related